Dockerfile

```diff
@@ -26,7 +26,6 @@ RUN apt-get update && \
     g++ \
     gnupg \
     gcc \
-    git \
```

Comment:
I like this being removed, but I thought it was required for some reason by fides?

Reply:
I actually added it in order to install the fideslang branch from git in this commit: https://github.com/ethyca/fides/pull/2343/commits/7a9cb45d41789f2c2f0f8a035f4e995e0923c20e


I reverted it the original install method in the most recent commit.